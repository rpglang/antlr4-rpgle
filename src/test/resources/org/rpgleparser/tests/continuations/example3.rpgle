     F someFile
     F                                     EXTIND
     F                                     (
     F                                     *INU1
     F                                     )