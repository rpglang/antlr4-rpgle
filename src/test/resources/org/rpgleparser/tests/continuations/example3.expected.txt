/*===TOKENS===*/
FS_FIXED                 <F>
FS_RecordName            < someFile >
FS_Type                  < >
FS_Designation           < >
FS_EndOfFile             < >
FS_Addution              < >
FS_Sequence              < >
FS_Format                < >
FS_RecordLength          <     >
FS_Limits                < >
FS_LengthOfKey           <     >
FS_RecordAddressType     < >
FS_Organization          < >
FS_Device                <       >
FS_Reserved              < >
KEYWORD_EXTIND           <EXTIND>
'('                      <(>
ID                       <*INU1>
')'                      <)>
/*===TREE===*/
(r
  (statement
    (fspec_fixed
      F
      ' someFile '
      ' '
      ' '
      ' '
      ' '
      ' '
      ' '
      '     '
      ' '
      '     '
      ' '
      ' '
      '       '
      ' '
      (fs_keyword
        (keyword_extind
          EXTIND
          (
          (simpleExpression (identifier (free_identifier (idOrKeyword *INU1))))
          )
        )
      )
      <EOF>
    )
  )
)
/*======*/